### ISSUE TITLE
big problem

### ISSUE THREAD

**Issue by funcieqDEV** [Ayush-Mann-0/custom-image-format#1]
its just a copy of spif-image-format, its not your code

**Comment by Ayush-Mann-0** [comment]
> its just a copy of spif-image-format, its not your code

yes i know its meant as personal project that im trying to learn rather than for others
